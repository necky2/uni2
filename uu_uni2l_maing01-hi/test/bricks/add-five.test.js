import UU5 from "uu5g04";
import AddFive from "../../src/bricks/add-five";

const { shallow } = UU5.Test.Tools;

describe(`UuUni2l.Bricks.AddFive`, () => {
  it(`default props`, () => {
    const wrapper = shallow(<AddFive />);
    expect(wrapper).toMatchSnapshot();
  });
  it(`number prop`, () => {
    const wrapper = shallow(<AddFive number={3} />);

    const renderedText = wrapper.text();
    expect(renderedText).toBe("3+5=8");

    wrapper.setProps({ number: 300 });

    const newRenderedText = wrapper.text();
    expect(newRenderedText).toBe("300+5=305");
  });
});
