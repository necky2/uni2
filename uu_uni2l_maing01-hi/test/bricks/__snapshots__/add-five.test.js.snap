// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UuUni2l.Bricks.AddFive default props 1`] = `
<div
  className="uuunil-bricks-0"
>
  0
  +
  5
  =
  5
</div>
`;
